File: euler in c/euler2.c
#include <stdio.h>

int main()
{
	int sum = 2;
	int x = 3;
	int x1 = 2;
	int x2 = 1; //x는 현재항 x1은 바로앞항 x2는 두개앞항
	while(x <= 4000000)
	{
		if (x%2==0)
		{
		sum += x;
		}
		printf ("%d \n", x);
		x = x1 + x2;
		x2 = x1;
		x1 = x;
	}
	
	printf("sum = %d", sum);
	return 0;
}

File: euler2/euler2.py
sequence = [1, 2]

number = input("몇 이하의 숫자까지 피보나치 수열을 진행할 것인지 결정하시오 : ")

i = 0
sum = 2
print(sequence[i])
print(sequence[i+1])
while sequence[i]+sequence[i+1] < int(number):
    sequence.append(sequence[i] + sequence[i+1])
    print(sequence[i+2])
    if sequence[i+2]%2==0:
        sum+=sequence[i+2]
        print("sum : %d" % sum)
    i+=1


print("result : %d" % sum)

